jsdoc/services/code-name-matchers/export-default-declaration.spec.js

```diff
@@ -2,13 +2,39 @@ var matcherFactory = require('./export-default-declaration');
 
 describe('ExportDefaultDeclaration matcher', function() {
 
-  var matcher;
+  var matcher, codeNameServiceMock;
 
   beforeEach(function() {
-    matcher = matcherFactory();
+    codeNameServiceMock = {
+      find: function (arg) {
+        return arg;
+      }
+    };
+    matcher = matcherFactory(codeNameServiceMock);
   });
 
-  it("should return null for any argument", function() {
-    expect(matcher({})).toBeNull();
+  it("should start search for right", function () {
+    var expr = {
+      right: 'right'
+    };
+
+    spyOn(codeNameServiceMock, 'find').and.callThrough();
+
+    expect(matcher(expr)).toEqual(expr.right);
+    expect(codeNameServiceMock.find.calls.count()).toEqual(1);
+    expect(codeNameServiceMock.find).toHaveBeenCalledWith(expr.right);
+  });
+
+  it("should return null for empty right", function () {
+    codeNameServiceMock.value = null;
+    var expr = {
+      right: null
```

Comment:
How about `right: ''` or `right: undefined` to show that we return `null` if the `right` is falsy?